mappings/net/minecraft/block/entity/TickAction.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_5558 net/minecraft/block/entity/TickAction
+	COMMENT An action that may be scheduled to run once per game tick per given block entity.
+	METHOD tick (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;Lnet/minecraft/class_2586;)V
+		COMMENT Runs this action on the given block entity. The world, block position, and block state are passed
+		COMMENT as context.
+		ARG 1 world
+		ARG 2 pos
+		ARG 3 state
+		ARG 4 blockEntity
```

Comment:
As of https://github.com/FabricMC/yarn/pull/1836 you might want to remove this, since it's now mapped (even though github doesn't show it)